hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/IncrSourceCloudStorageHelper.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.hudi.utilities.sources.helpers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.hudi.common.config.TypedProperties;
+import org.apache.hudi.common.util.Option;
+import org.apache.hudi.exception.HoodieException;
+import org.apache.hudi.utilities.sources.helpers.gcs.FileDataFetcher;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.spark.sql.DataFrameReader;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import static org.apache.hudi.common.util.StringUtils.isNullOrEmpty;
+import static org.apache.hudi.utilities.sources.helpers.gcs.GcsIngestionConfig.SPARK_DATASOURCE_OPTIONS;
+
+/**
+ * Helper methods for when the incremental source is fetching from Cloud Storage, like AWS S3 buckets or GCS.
+ */
+public class IncrSourceCloudStorageHelper {
+
+  private static final Logger LOG = LogManager.getLogger(FileDataFetcher.class);
```

Comment:
Change the class name in logger.

Reply:
Will do. Actually in the same folder I've found classes using LOG, logger, and log!  - https://github.com/apache/hudi/tree/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers

Reply:
Actually had to retain it as LOG because checkstyle complained!

Reply:
Ok, I was talking about className FileDataFetcher in the log line. That should be changed to IncrSourceCloudStorageHelper.

Reply:
Ack. Done.